rllib/agents/slateq/slateq.py

```diff
@@ -230,6 +230,11 @@ class SlateQTrainer(DQNTrainer):
     def get_default_config(cls) -> TrainerConfigDict:
         return SlateQConfig().to_dict()
 
+    @override(DQNTrainer)
```

Comment:
Why do we even need to override SlateQ validate_config here?
Don't we call `validate_buffer_config` in DQN as well and if yes, shouldn't DQN also validate the buffer config?

Reply:
Thanks, changed!